README.md

```diff
@@ -97,11 +100,12 @@ We support various module formats such as:
 
 ### 📌 Project usage
 
-* Install the dependency in your **package.json** file:
+Install the dependency in your **package.json** file:
 ```shell script
 npm i bpmn-visualization
 ```
 
+Put this JavaScript snippet in a JS file to initialize `bpmn-visualization` and to load the BPMN diagram:
```

Comment:
I still prefer
- Then use this JavaScript/TypeScript snippet to load your BPMN Diagram in a page
- Then use this snippet to load your BPMN Diagram in a page